etest/profile.py

```diff
@@ -0,0 +1,124 @@
+"""Profile definitions and builders."""
+
+import logging
+from enum import Enum
+
+
+class _arm_mappings(Enum):
+    armv5 = "armv5tel"
+    armv6 = "armv6j_hardfp"
+    armv7 = "armv7a_hardfp"
+    arm64 = "arm64"
+
+
+class InvalidProfileError(ValueError):
+    """The profile is invalid."""
+
+    pass
+
+
+class Profile:
+    """A class to generate a profile string and information.
+
+    Uses cli options.
```

Comment:
This isn't needed.  Just simplify to the header line of this docstring.

Reply:
Im sorry, I dont understand

Reply:
I was just saying drop the phrase "Uses cli options." from this docstring and make the docstring a single line.